CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.10.1
```

Comment:
Unfortunately, I think this needs to be released as 0.11 as it is changing the Public API (implementation of a non-fundamental trait).

Reply:
Following https://rust-lang.github.io/rfcs/1105-api-evolution.html#minor-change-implementing-any-non-fundamental-trait this is classified as a minor change, as it can "only" introduce ambiguity during method resolution, which I think is just never true here, since the impl is probably never called directly in user code anyway.

Reply:
I don't think "probably never" is a sound argument when it comes to following RFC.

Reply:
Still, the RFC says

> However, as before, this kind of breakage is considered "minor" by the principles of this RFC (see "Adding a defaulted item" above).